worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/brush/ClipboardBrush.java

```diff
@@ -42,16 +43,30 @@ public class ClipboardBrush implements Brush {
     public ClipboardBrush(ClipboardHolder holder, boolean ignoreAirBlocks, boolean usingOrigin) {
         this.holder = holder;
         this.ignoreAirBlocks = ignoreAirBlocks;
+        this.ignoreStructureVoidBlocks = true;
         this.usingOrigin = usingOrigin;
         this.pasteBiomes = false;
         this.pasteEntities = false;
         this.sourceMask = null;
     }
 
+    @Deprecated
     public ClipboardBrush(ClipboardHolder holder, boolean ignoreAirBlocks, boolean usingOrigin, boolean pasteEntities,
                           boolean pasteBiomes, Mask sourceMask) {
         this.holder = holder;
         this.ignoreAirBlocks = ignoreAirBlocks;
+        this.ignoreStructureVoidBlocks = true;
+        this.usingOrigin = usingOrigin;
+        this.pasteEntities = pasteEntities;
+        this.pasteBiomes = pasteBiomes;
+        this.sourceMask = sourceMask;
+    }
+
+    public ClipboardBrush(ClipboardHolder holder, boolean ignoreAirBlocks, boolean ignoreStructureVoidBlocks,
```

Comment:
the amount of params here is getting ugly but can we at least collapse these into properly overloaded constructors (`this(holder, ignoreAir...);`) so we reduce the repetition and re-setting of fields.

Reply:
We could even introduce a new `record` for these parameters, and have the canonical constructor be the record-taking one.